draft-ietf-tls-tls13.md

```diff
@@ -4375,6 +4375,91 @@ currently deployed and properly configured TLS implementations.
 %%### Ticket Establishment
 %%### Updating Keys
 
+# State Machine
+
+This section provides a summary of the legal state transitions for the
+client and server handshakes.  State names (in all capitals, e.g.,
+START) have no formal meaning but are provided for ease of
+comprehension.  Messages which are sent only sometimes are indicated
+in [].
+
+## Client
+
+~~~~
+                  +-----> START
+             Recv |        |
+            Hello |        | Send ClientHello
+            Retry |        v
+        +-> Request +---- WAIT_SH
```

Comment:
Something about the alignment here is a bit off. I think you lost two spaces on the first few lines here:

```
                    +-----> START
               Recv |        |
              Hello |        | Send ClientHello
              Retry |        v
        +-> Request +---- WAIT_SH
    Can |                    | Recv ServerHello
   Send |                    V
```

Hyphens might also be useful:

```
                    +-----> START
               Recv |        |
             Hello- |        | Send ClientHello
             Retry- |        v
        +-> Request +---- WAIT_SH
    Can |                    | Recv ServerHello
   Send |                    V
```

Or you can then avoid line-wrapping if you swap the branches:

```
                            START <---+
                             |        |
            Send ClientHello |        | Recv HelloRetryRequest
                             v        |
        +->               WAIT_SH ----+
    Can |                    | Recv ServerHello
   Send |                    V
```